lib/MqttClient.js

```diff
@@ -240,6 +242,8 @@ class MqttClient {
     updateAttributesTopicOnEvent(statusData) {
         this.last_ha_state = HA_STATE_MAPPINGS[statusData.state];
         this.last_state = statusData.state;
+        this.last_cleaned_area = statusData.clean_area ? (statusData.clean_area / 1000000).toFixed(1) : 0;
```

Comment:
```suggestion
        this.last_cleaned_area = statusData.clean_area ? parseFloat((statusData.clean_area / 1000000).toFixed(1)) : 0;
```
otherwise the data type is sometimes string, sometimes number

Reply:
Also we might increase the precision and leave it to the user consuming the MQTT to format accordingly

Reply:
Since these are rather imprecise floats, I don't think that that would make much sense to transmit data that will be certainly discarded anyways.

It's just ugly to look at random floating point imprecisions imho. Especially since the robot itself is around 0.1m² in size

Maybe .toFixed(2)? That would result in 10cm squares being the minimum resolution